pgcontents/tests/test_pgmanager.py

```diff
@@ -18,15 +18,18 @@
 from __future__ import unicode_literals
 
 from base64 import b64encode
+from cryptography.fernet import Fernet
```

Comment:
I think this was mentioned in Slack but wanted to make sure you didn't forget...do we need to add cryptography to requirements.txt or specify it as an optional requirement somehow?